fuel/transformers/__init__.py

```diff
@@ -590,3 +590,31 @@ def get_data(self, request=None):
         if data == StopIteration:
             raise StopIteration
         return data
+
+
+class RenameStream(Transformer):
+    """Renames the sources of the stream.
+
+    Parameters
+    ----------
+    data_stream : :class:`DataStream` or :class:`Transformer`
+        The data stream.
+    names : dict
+        A dictionnary mapping the old and new names of the sources to rename.
+
+    """
+    def __init__(self, data_stream, names):
+        super(RenameStream, self).__init__(data_stream)
+        sources = list(self.data_stream.sources)
+        for old, new in names.iteritems():
```

Comment:
Travis fails because `dict.iteritems` exists only in Python 2.x. Use [six.iteritems](https://pythonhosted.org/six/#six.iteritems) for 2/3 cross-compatibility.